src/assets/i18n/it.po

```diff
@@ -4328,37 +4395,37 @@ msgid "Delete Mountpoint"
 msgstr "Elimina Mountpoint"
 
 msgid "Delete Path"
-msgstr ""
+msgstr "Elimina Percorso"
 
 msgid "Delete Selected"
-msgstr ""
+msgstr "Elimina Selezionati"
 
 msgid "Delete User "
-msgstr ""
+msgstr "Elimina Utente"
 
 msgid "Delete Virtual Machine"
-msgstr ""
+msgstr "Elimina Macchina Virtuale"
 
 msgid "Delete Virtual Machine Data?"
-msgstr ""
+msgstr "Eliminare i Dati della Macchina Virtuale?"
```

Comment:
qua ci sarebbe da fare il discorso maiuscolo perché in inglese mettono tutti i titoli maiuscoli ma noi non lo facciamo... detto ciò meglio avere una traduzione che troppe finezze :)

Reply:
concordo! mettiamo in minuscolo a parte acronimi.

Mi viene da dire che forse anche i doppi spazi tra le parole nella singola frase non influiscono. Lascerei solo lo spazio finale e lunghezza linea.